src/processTranslation.spec.tsx

```diff
@@ -15,6 +15,8 @@ describe('processTranslation', () => {
       [ 'reviews', { count: 123 }, '123 reviews' ],
       [ 'boardBasis.code', {}, 'Board Basis' ],
       [ 'boardBasis.code', { context: 'AI' }, 'All Inclusive' ],
+      [ 'boardBasis.code', { context: 'AI', count: 1 }, 'All Inclusive' ],
+      [ 'boardBasis.code', { context: 'AI', count: 2 }, 'All Inclusive plural' ],
```

Comment:
`All Inclusive plural` lol what an example

Reply:
Changed to "All inclusive hotels"